src/components/ElementSearchDialog/element-search-dialog.js

```diff
@@ -105,7 +105,14 @@ const ElementSearchDialog = (props) => {
                     noOptionsText={intl.formatMessage({
                         id: 'element_search/noResult',
                     })}
-                    renderOption={renderElement}
+                    renderOption={(prop, element, { inputValue }) =>
+                        renderElement({
+                            props,
+                            element,
+                            inputValue,
+                            close: onClose,
```

Comment:
Why do you have to pass the onClose callback here ?

Reply:
because I need it ;)
in study app, to close the dialog when clicked